README.md

```diff
@@ -127,11 +127,12 @@ juju deploy .
 
 # Operator Framework development
 
-If you want to work in the framework *itself* you will need the following depenencies installed in your system:
+If you want to work in the framework *itself* you will need Python >= 3.5 and the dependencies declared in requirements-dev.txt installed in your system. Or you can use a virtualenv:
 
-- Python >= 3.5
-- PyYAML
-- autopep8
-- flake8
+```
+virtualenv --python=python3 env
+source env/bin/activate
+pip install -r requirements-dev.txt
+```
```

Comment:
I think making it an indented block is easier to read when not rendering it as markdown. Thus:
```suggestion
    virtualenv --python=python3 env
    source env/bin/activate
    pip install -r requirements-dev.txt
```